kusto/ingest/internal/resources/resources.go

```diff
@@ -45,8 +49,8 @@ type URI struct {
 	sas                             url.Values
 }
 
-// parse parses a string representing a Kutso resource URI.
-func parse(uri string) (*URI, error) {
+// Parse parses a string representing a Kutso resource URI.
+func Parse(uri string) (*URI, error) {
```

Comment:
I think the API can be tested without being made public

Reply:
its not for testing - i use it in streaming package now as well